src/standalone/Bucket/bucket_parameterizations.jl

```diff
@@ -55,20 +55,26 @@ a helper function which computes and returns the surface evaporative scaling
  factor for the bucket model.
 """
 function ClimaLand.surface_evaporative_scaling(model::BucketModel, Y, p)
-    beta = beta_factor.(Y.bucket.W, Y.bucket.σS, model.parameters.W_f)
+    beta =
+        beta_factor.(
+            Y.bucket.W,
+            Y.bucket.σS,
+            model.parameters.W_f,
+            model.parameters.σS_c,
```

Comment:
Just checking, do all of the params (incl. S_c and p) come from the SLIM model?

Reply:
SLIM uses the linear model we were using before for beta (beta = W/(0.75*Wf) for W < 0.75Wf, 1 otherwise). adding in the nonlinearity is our modification. They use W_f of 200 mm or 0.2m, sigmaS_c of 50 mm (0.05m). There is no separate beta for snow. They write:
```
Note that if the latent heat flux term attempts to evaporate (or sublimate)
139 more water than is available in the combined water and snow buckets, the latent heat flux term is
140 set to the total water plus snow available ... and the excess energy that would have
141 been used by LH if more water were available is instead partitioned to SH.
```

Other parameters:
volumetric heat capacity of soil is 2e6 J/m^3/K
soil K is 1.5 W/m/K
soil depth of 3.5m.

I think the main differences are:
- SLIM also uses an additional resistance to evaporation,
- SLIM modifies the MO fluxes for vegetation (vegetation height affects roughness lengths). 
- SLIM has some representation of glaciers

Reply:
Sounds great, thanks!